Issue 37 draft: Why it works vignette with analytic solutions (#68)
epinowcast/primarycensoreddist: vignettes/why-it-works.Rmd

---
title: "Interval censoring and truncation in epidemiological delay modelling"
format: html
bibliography: library.bib
---

# Introduction {#introduction}

> Time to event analysis, also known as survival analysis, concerns estimating the distribution of delay times between events. A distinctive feature of the field are the methodological techniques used to deal with the missing data problems common in data sets of delay times [@leung1997censoring]. In `primarycensoreddist` we focus on two particular missing data problems:

Reviewer: Here and later could do some referencing out to Park and Charniga for alternative explanations of various points.

Author: Yeah got that in there.

> -   **Interval censoring**. The primary (start) time and/or the secondary (end) time of the delay is unobserved but known to be within an interval.

Reviewer: ```suggestion
-   **Interval censoring**. The primary (start) time and/or the secondary (end) time of the delay are unobserved but both are known to be within intervals.
```

> -   **Truncation**. Delay data items are only contained in the data set conditional on aspect of the data point, for example if the secondary event is the observation time which add the data item to the data set.

Reviewer: ```suggestion
-   **Truncation**. Delay data items are only contained in the data set conditional on an aspect of the data point, for example, if the secondary event is the observation time which adds the data item to the data set.
```

I'm not totally clear what you were going for here and it doesn't totally make sense to me even with edits. I've left it for you vs writing a def from my view point as I think its helpful if it comes from your perspective vs repeating mine from i.e. the getting started and other papers etc.

In statistical epidemiology, these missing data problems occur frequently in both data analysis and theoretical modelling.

> In data analysis, events in epidemiology are commonly reported as occurring in a particular day or week (*interval censoring*). Also, in an emergency situation data analysis can be happening on datasets that are streaming in real time, which introduces biases around which data items have been observed (*truncation*).

Reviewer: ```suggestion
In data analysis, events in epidemiology are commonly reported as occurring on a particular day or week (*interval censoring*). In an emerging outbreak, datasets can be incompletely observed (*right truncation*) and their can be a great deal of uncertainty around the precise timing of events (*interval censoring*).
```

> In theoretical epidemiological modelling, it is often appropriate to model the evolution of an infectious disease as occurring in discrete time, for example in the [`EpiNow2`](https://epiforecasts.io/EpiNow2/) modelling package. This means appropriately discretizing continuous distributions, such as the generation interval distribution. In `primarycensoreddist` we treat the discretization of intrinsically continuous distributions as an *interval censoring* problem.

Reviewer: ```suggestion
In theoretical epidemiological modelling, it is often appropriate to model the evolution of an infectious disease as occurring in discrete time, for example in the [`EpiNow2`](https://epiforecasts.io/EpiNow2/) modelling package. This means appropriately discretising continuous distributions, such as the generation interval distribution. In `primarycensoreddist` we treat the discretisation of intrinsically continuous distributions as an *interval censoring* problem.
```

> In theoretical epidemiological modelling, it is often appropriate to model the evolution of an infectious disease as occurring in discrete time, for example in the [`EpiNow2`](https://epiforecasts.io/EpiNow2/) modelling package. This means appropriately discretising continuous distributions, such as the generation interval distribution. In `primarycensoreddist` we treat the discretisation of intrinsically continuous distributions as an *interval censoring* problem.

Reviewer: ```suggestion
In theoretical epidemiological modelling, it is often appropriate to model the evolution of an infectious disease as occurring in discrete time, for example in the [`EpiNow2`](https://epiforecasts.io/EpiNow2/) and [EpiEstim](https://mrc-ide.github.io/EpiEstim/index.html) modelling packages. This means appropriately discretising continuous distributions, such as the generation interval distribution. In `primarycensoreddist` we treat the discretisation of intrinsically continuous distributions as an *interval censoring* problem which allows us to simultaneously provide methods for both applied and theoretical contexts.
```

$$
f_P(t) \propto \exp(r t) \mathbb{1}_{[k, k + W_P]}(t).
$$

In this case, the distribution function for $C_P$, that is the length of time left in the primary censor window _after_ the primary event time, is given by:

$$
C(p) = {  1 - \exp(-r p) \over 1 - \exp(-r W_P)}, \qquad p \in [0, W_P].
$$

We consider two situations: 1) uniform primary event time ($r=0$) and 2) non-uniform primary event time ($r\neq0$).

### Uniform primary event time ($r=0$)

In this case, the primary event time is uniformly distributed within the primary censoring window. The survival function of $S_+$ is then:

$$
Q_{S_+}(t) = Q_T(t + W_P) + { 1 \over W_P} \int_0^{W_P} f_T(t+p) p~ dp.
$$

> This is analytically solvable whenever the upper distribtution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.

Reviewer: ```suggestion
This is analytically solvable whenever the upper distribution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.
```

In theoretical epidemiological modelling, it is often appropriate to model the evolution of an infectious disease as occurring in discrete time, for example in the [`EpiNow2`](https://epiforecasts.io/EpiNow2/) modelling package. This means appropriately discretising continuous distributions, such as the generation interval distribution. In `primarycensoreddist` we treat the discretisation of intrinsically continuous distributions as an *interval censoring* problem.

# Statistical model used in `primarycensoreddist`

As described in [Introduction](#introduction), `primarycensoreddist` focuses on a subset of methods from time to event analysis that address data missingness problems commonly found in epidemiological datasets. We present the statistical problem as a double interval censoring problem, where both the primary event time and the secondary event times are interval censored. We can recover single interval censoring problems by reducing one of the intervals to a point.

## Survival function of secondary event time after primary event window

> A key assumption we make is that the censoring window for events is known and independent of the event time. This is known as non-informative censoring. It is useful to consider the time from the end (right) point of the primary censoring interval to the secondary time as a random variable,

Reviewer: define S_+ and thee interval notation more generally i.e S, P upper and lower bounds etc

Reviewer: helpful to tell people about CDF and survival a little more clearly

$$
S_+ = T - C_P.
$$

Where $T$ is the delay distribution of interest and $C_P$ is interval between the end (right) point of the primary censoring window and the primary event time; note that by definition $C_P$ is not observed.

> With non-informative censoring, it is possible to derive the upper distribution function of $S_+$, or _survival function_ of $S_+$, from the distribution of $T$ and the distribution of $C_P$.

Reviewer: some kind of reference of further hand holding is needed her I think

$$
Q_{S_+}(t) = \mathbb{E}_{C_P} \Big[Q_T(t + C_P)\Big] = \int_0^{W_P} Q_T(t + p) c(p) dp.
$$

> Using integration by parts gives:

Reviewer: give a link out to this (yes I know). I think Wikipedia is fine

Reviewer: also as a above we want to bring people with us so a little more exposition.

$$

> Q_{S_+}(t) = Q_T(t + W_P) + \int_0^{W_P} f_T(t+p) C(p) dp.

Reviewer: missing f_T def

$$

> Where $c$ is the density function of delay from the primary event to the end (right) point of the primary censoring window with distribution function $C$, $Q_T$ is the survival function of the actual delay distribution of interest and $W_P$ is the length of the primary censoring window.

Reviewer: To link across with the docs I think here we should connect it to the formulation give in the getting started clearly and the same goes for the pmf section below

## Probability mass of secondary event time within a secondary censoring window

The probability of the secondary event time falling within a secondary censoring window of length $W_S$ that begins at time $n$ _after_ the primary censoring window is then given by:

$$

> Pr(S_+ \in [n, n + W_S)) = Q_{S_+}(n) - Q_{S_+}(n + W_S).

Reviewer: have you got this the right way around?

Author: Yeah, its the probability of being greater than $n$ less the probability of being greater than $n+W_S$.

$$

Which can in general be calculated by numerical quadrature.

> Note that the secondary event time can also occur within the primary censoring window. This happens with probability,

Reviewer: link to getting started

> Note that the secondary event time can also occur within the primary censoring window. This happens with probability,

Reviewer: this currently feels like somewhat of a random aside. If it is can it go after the main thrust of your point and some text added to explain why we care.

$$
Q_{S_+}(-W_P) - Q_{S_+}(0) = 1 - Q_T(W_P) - \int_0^{W_P} f_T(p) C(p) dp = Pr(T< C).
$$

> Using a difference operator defined as:

Reviewer: What does this gain you apart from 74? line 74 doesn't seem that useful?

Author: It pops up a lot in the analytic solutions, I wasn't going to add new notation but then I realised that for the uniform primary censor window to basic delay pmf comes in a standard form.

Reviewer: maybe introduce it in the analytical section then (perhaps even as a helpful ideas section or similar)

$$
\Delta_{W}f(t) = f(t + W) - f(t).
$$

The common case where every primary and secondary event window is a single time unit, $W_P = W_S = 1$, simplifies the above expressions to:

$$
\begin{aligned}
f_n = Pr(S_+ \in [n, n + 1)) &= Q_{S_+}(n) - Q_{S_+}(n + 1), \\
& = - \Delta_1 Q_{S_+}(n),\qquad n = -1, 0,\dots.
\end{aligned}
$$

> Which is a discrete probability mass function of the secondary event time relative to the primary event time in units of the event window.

Reviewer: As above I think it might be nice to write this out in terms of Q_t and co (i.e the solution to Q_S+).

> 

Reviewer: We put a lot of effort in to tell people about truncation and now its a secret. I thnk this might work best as a standalone section which shows how truncation fits into the above ideas (i.e. its like any other truncation approach and all thee special clever SamB sauce can be ignored at this stage).

> ## Analytic solutions for exponentially tilted primary event times

Reviewer: As I mentioned elsewhere I think these could be their own vignette with a bit of intro linking back to why it works and some context before diving in.

Author: Agreed, I've got an issue for that.

> In epidemiological analysis, it is common for primary events to be arriving at exponentially increasing or decreasing rates, for example, incidence of new infections in a growing epidemic. In this case, the distribution of the primary event time within its censoring window is biased by the exponential growth or decay. If we assume a reference uniform distribution within a primary censoring window $[k, k + W_P)$ then the distribution of the primary event time within the censoring window is the exponential tilted uniform distribution:

Reviewer: link to our implementation? (i.e code docs [rexpgrowth()].

Reviewer: add ref to exponetial titled uniform

> In epidemiological analysis, it is common for primary events to be arriving at exponentially increasing or decreasing rates, for example, incidence of new infections in a growing epidemic. In this case, the distribution of the primary event time within its censoring window is biased by the exponential growth or decay. If we assume a reference uniform distribution within a primary censoring window $[k, k + W_P)$ then the distribution of the primary event time within the censoring window is the exponential tilted uniform distribution:

Reviewer: bit more on what this means (i.e no circadian pattern etc)

$$
f_P(t) \propto \exp(r t) \mathbb{1}_{[k, k + W_P]}(t).
$$

In this case, the distribution function for $C_P$, that is the length of time left in the primary censor window _after_ the primary event time, is given by:

$$
C(p) = {  1 - \exp(-r p) \over 1 - \exp(-r W_P)}, \qquad p \in [0, W_P].
$$

> We consider two situations: 1) uniform primary event time ($r=0$) and 2) non-uniform primary event time ($r\neq0$).

Author: Sure. That makes sense. Mostly, its not hard to extend solutions to situation 2) except that for log-normal dist where the maths is really helped by cancelling the $\frac{1}{z}$ factor in the pdf.

### Uniform primary event time ($r=0$)

In this case, the primary event time is uniformly distributed within the primary censoring window. The survival function of $S_+$ is then:

$$
Q_{S_+}(t) = Q_T(t + W_P) + { 1 \over W_P} \int_0^{W_P} f_T(t+p) p~ dp.
$$
This is analytically solvable whenever the upper distribution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.

In each case it is easier to change the integration variable:

$$
\begin{aligned}
Q_{S_+}(t) &= Q_T(t + W_P) + { 1 \over W_P} \int_t^{t+W_P} f_T(z) (z-t)~ dz \\

> &= Q_T(t + W_P) + { 1 \over W_P} \Big[  \int_t^{t+W_P} f_T(z) z~ dz - t \Delta_{W_P}F_T(t) \Big].

Reviewer: by parts blah blah

\end{aligned}
$$

> 

Reviewer: some text saying we can solve above for a bunch of dists and now lets go

#### Gamma distribution

The Gamma distribution has the density function:

$$

> f_T(z;k, \theta) = {1 \over \Gamma(k) \theta^k} z^{k-1} \exp(-z/\theta).

Reviewer: Also give F (and in conjunction with below use directly in the last statement you derived)

$$

This gives a solution to the survival function of $S_+$ in terms of analytically available functions:

$$
\begin{aligned}
Q_{S_+}(t; k, \theta) &= Q_T(t + W_P; k, \theta) + { 1 \over W_P} \left( {1 \over \Gamma(k) \theta^k} \int_t^{t+W_P} z^{k-1}  \exp(-z/\theta) (z-t)~ dz \right), \\
&= Q_T(t + W_P; k, \theta) + { 1 \over W_P} \big[ k \theta \Delta_{W_P}F_T(t; k+1, \theta) - t \Delta_{W_P}F_T(t; k, \theta) \big].
\end{aligned}
$$

> Where we have used the standard Gamma integration trick of rewriting

Reviewer: ref (wikipedia)

$$
{z z^{k-1} \over \Gamma(k) \theta^k} = {k \theta z^k \over\Gamma(k+1) \theta^{k+1} }.
$$

In the special case of the equal event window then the discrete delay distribution is:

$$
\begin{aligned}

> f_n &= (n+2) F_T(n+2; k, \theta) + n F_T(n; k, \theta) - 2  (n+1) F_T(n+1; k, \theta) \\

Reviewer: I think a few more steps here for people would be great.

 &+  k \theta \Big( 2 F_T(n+1; k+1, \theta) - F_T(n; k+1, \theta) - F_T(n+2; k+1,\theta)  \Big).
\end{aligned}
$$

Which was also found by Cori _et al_ [@cori2013new].

#### Log-Normal distribution

The Log-Normal distribution has the density function:

$$
f_T(z;\mu, \sigma) = {1 \over z \sigma \sqrt{2\pi}} \exp\left( - {(\log(z) - \mu)^2 \over 2 \sigma^2} \right).
$$
And distribution function:

$$

> F_T(z;\mu, \sigma) = \Phi\left( {\log(z) - \mu \over \sigma} \right).

Reviewer: i.e do this above for the gamma

$$
Where $\Phi$ is the standard normal distribution function.

This gives a solution to the survival function of $S_+$ in terms of analytically available functions:

$$
\begin{aligned}
Q_{S+}(t ;\mu, \sigma) &= Q_T(t + W_P;\mu, \sigma) + { 1 \over W_P} \int_t^{t+W_P} {1 \over z \sigma \sqrt{2\pi}} \exp\left( - {(\log(z) - \mu)^2 \over 2 \sigma^2} \right) (z-t)~ dz, \\

> &= Q_T(t + W_P;\mu, \sigma) + { 1 \over W_P} \Big[ e^{\mu + \frac{1}{2} \sigma^2} \Delta_{W_P}F_T(t; \mu + \sigma^2, \sigma) - t\Delta_{W_P}F_T(t; \mu, \sigma) \Big]

Reviewer: just go directly to this line and say subbing into blah blah previous equation

\end{aligned}
$$

> Where we have used the Log-Normal integration trick of making a further substitution $y = (\ln z - \mu) / \sigma$ and using the shift property of the standard normal distribution function. This gives:

Reviewer: nice. cite

Reviewer: also subbing into to equation blah blah blah

Author: Yeah, I don't actually know a citation for this. I put "The log-normal integration trick" as a bit of a placeholder. I'll try and find a ref, its pretty obvs once you spot it.

&= \int_0^{w_P} Q_T(t + p) f_{C_P}(p) dp.
\end{split}
\end{equation}
$$

Using [integration by parts](https://en.wikipedia.org/wiki/Integration_by_parts) gives:
$$
Q_{S_+}(t) = Q_T(t + w_P) + \int_0^{w_P} f_T(t+p) F_{C_P}(p) dp. (\#eq:survivalfunc)
$$

Where we have used that $Q^{'}_{T} = - f_T$, $Q_T$ is the survival function of the actual delay distribution of interest and $w_P$ is the length of the primary censoring window.

Equation \@ref(eq:survivalfunc) is the key equation in this note and is used to derive the distribution of the censored delay time $T_c$. It has the interpretation that the probability that the secondary event time is greater than $t$ after the end of the primary censoring window is the sum of two disjoint event probabilities:

1. The probability that the _actual_ delay time $T$ is greater than $t + w_P$.
2. The probability that the _actual_ delay time $T$ is between $t$ and $t + w_P$, and the primary event time $P$ occurred sufficiently close to the end of the primary censor window that the secondary even occurred more than time $t$ after the end of the primary window.

### Probability of secondary event time within a secondary censoring window

> Having constructed the survival function of $S_+$ with equation \@ref(eq:survivalfunc), using numerical quadrature or in some other way, we can calculate the probability mass of a secondary event time falling within a observed secondary censoring window of length $w_S$ that begins at time $n$ _after_ the primary censoring window. This gives the censored delay time probability [by integrating over censored values](https://mc-stan.org/docs/2_18/stan-users-guide/censored-data.html):

Reviewer: ```suggestion
Having constructed the survival function of $S_+$ with equation \@ref(eq:survivalfunc), using numerical quadrature or in some other way, we can calculate the probability mass of a secondary event time falling within a observed secondary censoring window of length $w_S$ that begins at time $n$ _after_ the primary censoring window. This gives the censored delay time probability [by integrating over censored values](https://mc-stan.org/docs/stan-users-guide/truncation-censoring.html#integrating-out-censored-values):
```

up to date link

$$
F_{C_P}(p; r) = {  1 - \exp(-r p) \over 1 - \exp(-r w_P)}, \qquad p \in [0, w_P]. (\#eq:fcp)
$$
Note that taking the limit $\lim_r \rightarrow 0$ in equation \@ref(eq:fcp) gives the uniform distribution function $F_{C_P}(p, 0) = p / w_P$.

In the following it is convenient to use a (linear) difference operator defined as:

$$
\Delta_{w}f(t) = f(t + w) - f(t).
$$

### Uniform primary event time ($r=0$)

Applying a uniform primary event time distribution to equation \@ref(eq:survivalfunc) gives:

$$
Q_{S_+}(t) = Q_T(t + w_P) + { 1 \over w_P} \int_0^{w_P} f_T(t+p) p~ dp.
$$
This is analytically solvable whenever the upper distribution function of $T$ is known and the mean of $T$ is analytically solvable from its integral definition.

In each case considered below it is easier to change the integration variable:

$$
Q_{S_+}(t) = Q_T(t + w_P) + { 1 \over w_P} \int_t^{t+w_P} f_T(z) (z-t)~ dz = Q_T(t + w_P) + { 1 \over w_P} \Big[  \int_t^{t+w_P} f_T(z) z~ dz - t \Delta_{w_P}F_T(t) \Big]. (\#eq:unifprim)
$$

**General partial expectation**

> Note that for any distribution with an analytically available distribution function $F_T$ equation \@ref{eq:unifprim} can be solved so long as the _partial expectation_

Reviewer: ```suggestion
Note that for any distribution with an analytically available distribution function $F_T$ equation \@ref(eq:unifprim) can be solved so long as the _partial expectation_
```

$$
\int_t^{t+w_P} f_T(z) z~ dz (\#eq:partexp)
$$

can be reduced to an analytic expression.

**General Discrete censored delay distribution**

> First we note that equation \@ref{eq:disccensprob} can be written using the difference operator: $f_n = -\Delta_1 Q_{S_+}(n-1)$. We can insert this expression into equation \@ref{eq:unifprim} to give the discrete censored delay distribution for uniformly distributed primary event times:

Reviewer: ```suggestion
First we note that equation \@ref(eq:disccensprob) can be written using the difference operator: $f_n = -\Delta_1 Q_{S_+}(n-1)$. We can insert this expression into equation \@ref(eq:unifprim) to give the discrete censored delay distribution for uniformly distributed primary event times:
```

$$
f_n = \Delta_1\Big[(n-1) \Delta_1F_T(n-1)\Big] - \Delta_1Q_T(n) + \Delta_1\Big[ \int_{n-1}^n f_T(z) z ~dz \Big] = (n+1)F_T(n+1)  + (n-1)F_T(n-1) - 2nF_T(n) + \Delta_1\Big[ \int_{n-1}^n f_T(z) z ~dz \Big]. (\#eq:disccensunifprim)
$$

We now consider some specific delay distributions.

#### Gamma distributed delay times

The Gamma distribution has the density function:

$$
f_T(z;k, \theta) = {1 \over \Gamma(k) \theta^k} z^{k-1} \exp(-z/\theta).
$$
Where $\Gamma$ is the Gamma function.

The Gamma distribution has the distribution function:

$$
\begin{aligned}
F_T(z;k, \theta) &= {\gamma(k, z/\theta) \over \Gamma(k)}, \qquad z\geq 0,\\
F_T(z;k, \theta) &= 0, \qquad z < 0.
\end{aligned}
$$
Where $\gamma$ is the lower incomplete gamma function.

**Gamma partial expectation**

$$
\begin{aligned}
\int_t^{t+w_P} f_T(z) z~ dz &= {1 \over \Gamma(k) \theta^k} \int_t^{t+w_P}z z^{k-1} \exp(-z/\theta)~dz \\
&=  {\Gamma(k+1) \theta^{k+1} \over \Gamma(k) \theta^k}  {1 \over \Gamma(k+1) \theta^{k+1}} \int_t^{t+w_P}z^{k} \exp(-z/\theta)~dz\\
&= k\theta \Delta_{w_P} F_T(t; k + 1, \theta).
\end{aligned} (\#eq:gammapartexp)
$$

**Survival function of $S_{+}$ for Gamma distribution**

> By substituting equation \@ref{eq:gammapartexp} into equation \@ref{eq:disccensunifprim} we can solve for both the survival function of $S_+$ in terms of analytically available functions:

Reviewer: ```suggestion
By substituting equation \@ref(eq:gammapartexp) into equation \@ref(eq:disccensunifprim) we can solve for both the survival function of $S_+$ in terms of analytically available functions:
```

$$
\begin{aligned}

> Q_{S_+}(t; k, \theta) &= Q_T(t + w_P; k, \theta) + { 1 \over w_P} \left( {1 \over \Gamma(k) \theta^k} \int_t^{t+w_P} z^{k-1}  \exp(-z/\theta) (z-t)~ dz \right), \\

Reviewer: do you think we actually need the first line or is by subsitutation and the second line just enough? In some sense it feels like scary maths that isn't actually adding anything now?

#### Log-Normal distribution

The Log-Normal distribution has the density function:

$$
f_T(z;\mu, \sigma) = {1 \over z \sigma \sqrt{2\pi}} \exp\left( - {(\log(z) - \mu)^2 \over 2 \sigma^2} \right).
$$
And distribution function:

$$
F_T(z;\mu, \sigma) = \Phi\left( {\log(z) - \mu \over \sigma} \right).
$$
Where $\Phi$ is the standard normal distribution function.

**Log-Normal partial expectation**

We use an integration substitution $y = (\ln z - \mu) / \sigma$ and using the shift property of the standard normal distribution function. This gives:

$$
\begin{aligned}
\int_t^{t+w_P} z~ f_T(z; \mu, \sigma) dz &= {1 \over \sigma \sqrt{2\pi}} \int_{(\ln t - \mu)/\sigma}^{(\ln(t+w_P) - \mu)/\sigma} e^{\sigma y + \mu} e^{-y^2/2} dy\\
&= e^{\mu + \frac{1}{2} \sigma^2} \int_{(\ln t - \mu)/\sigma}^{(\ln(t+w_P) - \mu)/\sigma} e^{-(y- \sigma)^2/2} dy \\
&= e^{\mu + \frac{1}{2} \sigma^2} \Big[\Phi\Big({\ln(t+w_P) - \mu \over \sigma} - \sigma\Big) - \Phi\Big({\ln(t) - \mu \over \sigma} - \sigma\Big) \Big]\\
&= e^{\mu + \frac{1}{2} \sigma^2} \Delta_{w_P}F_T(t; \mu + \sigma^2, \sigma).
\end{aligned} (\#eq:lognormpartexp)
$$

**Survival function of $S_{+}$ for Log-Normal distribution**

> By substituting equation \@ref{eq:lognormpartexp} into equation \@ref{eq:disccensunifprim} we can solve for both the survival function of $S_+$ in terms of analytically available functions:

Reviewer: ```suggestion
By substituting equation \@ref(eq:lognormpartexp) into equation \@ref(eq:disccensunifprim) we can solve for both the survival function of $S_+$ in terms of analytically available functions:
```

---
title: "Interval censoring and truncation in epidemiological delay modelling"

> author: "Samuel Brand"

Reviewer: This is fine (though perhaps package wide we want to not have authors as it becomes unclear when more people start contributing) but I actually meant to the package DESCRIPTION.

Author: That makes sense!
